### ISSUE TITLE
stable v1.5?

### ISSUE THREAD

**Issue by balage06** [baffler/OBS-OpenVR-Input-Plugin#48]
Hi!
Is the v1.5 version safe to use? It's tagged as pre-release, but it sounds useful based on the changelog.

**Comment by baffler** [comment]
Yeah, I was tagging it as pre-release cause there might be bugs with the new feature. I do believe there is 1 bug that I've noticed with the cropping but it's nothing major.

**Comment by Gareds** [comment]
And this  version  not  working  with OBS  :D (but work  with StreamLabsOBS).
Everything is  correct. No mistakes  on  my  side. So its  a  bug  :)
